terraform/parser.go

```diff
@@ -32,14 +32,18 @@ type NewParserParams struct {
 
 type JSON = map[string]interface{}
 
+func NewParserVariables(variables ModuleVariables) ValueMap {
+	return ValueMap{
+		"var":   cty.ObjectVal(variables.inputs),
+		"local": cty.ObjectVal(variables.locals),
+	}
+}
+
```

Comment:
was this extraction to help getting it used in dereferenceLocals()?

Reply:
The purpose of this extraction is to help us configure the variables map that we are using in our parser.